templates/email.yaml.erb

```diff
@@ -0,0 +1,15 @@
+# Outgoing email settings
+
+production:
+  delivery_method: <%= @email_delivery_method %>
```

Comment:
Not sure what erb does with undef here (and foreman if the value is empty/invalid). Maybe move the if above ```production:```?